data_ingestion/common/publisher.py

```diff
@@ -0,0 +1,25 @@
+from google.cloud import pubsub_v1
+
+
+def notify_data_ingested(id, gcs_bucket, filename):
+  """Publishes a notification on the notify-data-ingested topic indicating that
+     some data was ingested.
+
+     id: The id of the source that was ingested
+     gcs_bucket: The name of the bucket the data is located in
+     filename: The name of the file that was uploaded"""
+  publisher = pubsub_v1.PublisherClient()
+  # For some reason topic_path doesn't show up as a member, but is used in the official documentation:
+  # https://googleapis.dev/python/pubsub/latest/publisher/api/client.html?highlight=topic_path#google.cloud.pubsub_v1.publisher.client.Client.publish
+  # pylint: disable=no-member
+  topic_path = publisher.topic_path('fellowship-test-internal', 'notify-data-ingested')
+
+  # Not sure if anything here is necessary since we can add attributes directly.
+  # For now just adding a message to log.
+  data = 'Notifying data ingested'
+  data = data.encode('utf-8')
+  future = publisher.publish(
+      topic_path, data, id=id, gcs_bucket=gcs_bucket, filename=filename
+  )
+  # TODO: Error handling for result
+  print(future.result())
```

Comment:
Is there a way to check the future for success/failure and use the `logging` api here?

Reply:
I'm 99% sure there is, I'll look into adding that